Test.C

```diff
@@ -255,7 +275,7 @@ int main(){
         // Present attack
         // print attack details and affects of attack
         // edit encrypt structure effectiveness
-        if (rounds[currentRound].attack > -1){
+        if (rounds[currentRound].attack == rounds[previousRound].attack){ //changed this portion becuase it would not work after I added the random rounds
```

Comment:
```suggestion
        if (rounds[currentRound].attack > -1){ 
```

Was preventing the execution of any attacks.